src/vmm/src/devices/virtio/balloon/mod.rs

```diff
@@ -92,20 +92,25 @@ pub enum BalloonError {
     StatisticsStateChange,
     /// Amount of pages requested cannot fit in `u32`.
     TooManyPagesRequested,
-    /// Error while processing the virt queues.
+    /// Error while processing the virt queues: {0}
     Queue(QueueError),
-    /// Error removing a memory region at inflate time.
+    /// Error removing a memory region at inflate time: {0}
     RemoveMemoryRegion(RemoveRegionError),
-    /// Error creating the statistics timer.
+    /// Error creating the statistics timer: {0}
     Timer(std::io::Error),
 }
 
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error, displaydoc::Display)]
 pub enum RemoveRegionError {
+    /// AddressTranslation
     AddressTranslation,
+    /// MalformedRange
```

Comment:
nit: For these too, can we the message a bit more descriptive

Reply:
fixed